File: src/test/java/no/nav/fo/veilarbregistrering/config/UnleashConfig.java
package no.nav.fo.veilarbregistrering.config;

import no.nav.common.featuretoggle.UnleashClient;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import static org.mockito.Mockito.mock;

@Configuration
public class UnleashConfig {

    @Bean
    public UnleashClient unleashClient() {
        return mock(UnleashClient.class);
    }

}

File: src/test/java/no/nav/fo/veilarbregistrering/config/UnleashConfig.kt
package no.nav.fo.veilarbregistrering.config

import io.mockk.mockk
import no.nav.common.featuretoggle.UnleashClient
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class UnleashConfig {
    @Bean
    fun unleashClient(): UnleashClient = mockk(relaxed = true)
}
